src/bundle.js

```diff
@@ -96,3 +96,8 @@ var libass_wasm = require("libass-wasm");
 _define("JavascriptSubtitlesOctopus", function() {
     return libass_wasm;
 });
+
+var queryString = require("query-string");
```

Comment:
dependencies missing

Reply:
Still a draft, I didn't commit my package.json since I added a few things to eslint to help track down stuff I need to cleanup :)